tests/integration_tests/build/test_style.py

```diff
@@ -76,6 +76,16 @@ def test_python_style():
     )
 
 
+def test_rust_clippy():
+    """Fails if clippy generates any error, warnings are ignored."""
+    run(
+        'cargo clippy --all-targets --all-features',
```

Comment:
Please also use `-D warnings` flag. This will help in not allowing clippy warnings in the future.

The line should look like:
'cargo clippy --all-targets --all-features -- -D warnings'